### PR TITLE
Changes tajaran random name generator

### PR DESCRIPTION
**Pull request by Alberyk**
This pr changes the old bay method, replacing them with some arabic and slavic names instead. Because"Rrhazkrajri Njarir'Akhan" and the like sounds stupid and has some issues with the lore, because you are not named after your ethnicity.

### CODE REVIEW

**Comment by Lohikar** [comment]
Perhaps the name gen should be changed to read from a file like the human one?

**Comment by LordFowl** [comment]
Not really necessary. Both the Dionaea and Vaurcae name generators work fine from in-code. The only reason humans could possibly warrant a seperate files is because of how many possibilities there are.

File: code/modules/mob/language/station.dm [hunk 1]
```diff
@@ -56,11 +56,13 @@
...
 /datum/language/tajaran/get_random_name(var/gender)
 
-	var/new_name = ..(gender,1)
-	if(prob(50))
-		new_name += " [pick(list("Hadii","Kaytam","Nazkiin","Zhan-Khazan","Hharar","Njarir'Akhan","Faaira'Nrezi","Rhezar","Mi'dynh","Rrhazkal","Bayan","Al'Manq","Mi'jri","Chur'eech","Sanu'dra","Ii'rka"))]"
-	else
-		new_name += " [..(gender,1)]"
+	var/new_name
+	if(gender==FEMALE)
+		new_name += " [pick(list("Aisha","Amani","Agashka","Agramakova","Bakhteiarova","Bratomila","Dekhova","Derzhka","Farah","Faizah","Hanifa","Ikram","Jenna","Khadija","Kandaza","Karyme","Kismet","Malika","Mumtaz","Nashwa","Nafisha","Shahar","Sabah","Takiyah","Wafiqah","Wasime","Yesenia","Zahra","Zohra","Rajjurl","Marryam","Dhrarmela","Rrhuyala","Irryavga","Ahvulyar","Purryalna","Yasmin"))]"
```
Comment (skull132) [inline code/modules/mob/language/station.dm]: I will never understand why people are silly like this.

```
new_name += "[pick(list("strings"))]"
```

Is the same as
```
new_name += pick(list("strings"))
```

IIRC. Should test.

File: code/modules/mob/language/station.dm [hunk 2]
```diff
@@ -56,11 +56,13 @@
...
+	else if(gender==MALE)
+		new_name += " [pick(list("Abdal","Akhtar","Akim","Alik","Arif","Azzam","Anton","Alzrun","Bahir","Barakh","Bohdan","Coman","Dekel","Dima","Fa'iq","Faidah","Fredek","Fakhr","Hasam","Hashim","Ioakim","Kadir","Kareef","Khrazmrrug","Khazarir","Rrhazumar","Mrrhazugal","Harrmalik","Mata'ke","Njadrasanukii","Tajrhalrr","Rhagrrhuzau","Vahzirthaamro","Lev","Misha","Mahir","Nasser","Nasir","Oleg","Qadir","Sharif","Stepka","Tamir","Yasir","Zahid"))]"
+	new_name += "[pick(list(" "))]"
```
Comment (skull132) [inline code/modules/mob/language/station.dm]: `new_name += " "`

**Comment by skull132** [comment]
Also, get this cleared with Mofo. Just in case.

**Comment by Alberyk** [comment]
Changes done, and Mofo is already aware, asked for some names to be added and approved of this.

**Pull request merged**